apps/judicial-system/backend/src/app/modules/notification/civilClaimantNotification.service.ts

```
@@ -0,0 +1,176 @@
+      switch (notificationType) {
+        case CivilClaimantNotificationType.SPOKESPERSON_ASSIGNED:
+          return this.sendSpokespersonAssignedNotification(
+            civilClaimant,
+            theCase,
+          )
+        default:
+          throw new InternalServerErrorException(
+            `Invalid notification type: ${notificationType}`,
+          )
+      }
+    }
```

Biome

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)